src/main/java/locustTask/AbstractYesNoApiTask.java

```diff
@@ -0,0 +1,36 @@
+package locustTask;
+
+import com.github.myzhan.locust4j.AbstractTask;
+import com.github.myzhan.locust4j.Locust;
+import io.restassured.RestAssured;
+import io.restassured.http.Method;
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import org.junit.Assert;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public abstract class AbstractYesNoApiTask extends AbstractTask {
+
+    private static final Logger logger = LoggerFactory.getLogger(YesNoApi.class);
+
+    protected static final String baseURI = "https://yesno.wtf/api";
+
+    protected void commonTask(String URLGetParam, String expected) {
+        Response response = null;
```

Comment:
Info: this is a demo code, in an ideal framework we should have a ServiceObject with the methods to be called.

Reply:
Hey! Understood.

Let me plan something and I'll come back again with a new proposal.

Thanks for the explanation.

Reply:
I have pushed a new proposal. The idea of SubTasks. You get the definition of all subtasks to be executed from any source (in this case a JSON file) and then one only task loops and executes every subtask.